cli/tests/test_windows_release_certification.py

```
@@ -121,15 +137,58 @@ def _run_private_custody_assertion(path: Path) -> subprocess.CompletedProcess[st
+        "$ErrorActionPreference = 'Stop'\n"
+        "Set-StrictMode -Version Latest\n"
+        f"{function}\n"
+        "$record = [Environment]::GetEnvironmentVariable('DC_TEST_JSON') | "
+        "ConvertFrom-Json\n"
+        "$propertyName = [Environment]::GetEnvironmentVariable('DC_TEST_PROPERTY')\n"
+        "$values = @(Get-OptionalJsonPropertyValue "
+        "-InputObject $record -PropertyName $propertyName)\n"
+        "[Console]::Out.WriteLine('count={0}' -f $values.Count)\n"
+        "[Console]::Out.WriteLine('value=<{0}>' -f ($values -join ','))\n"
+    )
+    return subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)